Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/serverexplore/hdinsightnode/StorageAccountNode.java

```diff
@@ -62,10 +62,14 @@ protected void loadAdditionalActions() {
         if (clusterDetail instanceof ClusterDetail) {
             addAction("Open Storage in Azure Management Portal", new NodeActionListener() {
                 @Override
-                protected void actionPerformed(NodeActionEvent e) throws AzureCmdException {
+                protected void actionPerformed(NodeActionEvent e) {
                     String subscriptionId = clusterDetail.getSubscription().getSubscriptionId();
                     String storageRelativePath = ((ClusterDetail) clusterDetail).getId() + REST_SEGMENT_STORAGE_ACCOUNT;
-                    openResourcesInPortal(subscriptionId, storageRelativePath);
+                    try {
```

Comment:
Why not put `try` after line 65

Reply:
Only `openResourcesInPortal` throws `AzureCmdException`, refer to https://github.com/microsoft/azure-tools-for-java/blob/6d10763058e4b6c2767f3ca4ff3dfb019c3ac9ab/Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/Node.java#L403 please.